src/EIPTestsFiller/StateTests/stEIP5656-MCOPY/MCOPY_memory_expansion_costFiller.yml

```diff
@@ -0,0 +1,221 @@
+MCOPY_memory_expansion_cost:
+
+  _info:
+    comment: Test cases for the memory expansion cost in the MCOPY instruction
+
+  env:
+    currentNumber: 1
+    currentTimestamp: 1687174231
+    currentGasLimit: 1000000
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+
+    # Executed code (transaction destination)
+    000000000000000000000000000000000000c0de:
+      balance: 0
+      nonce: 1
+      code: |
+        :yul {
+          function mcopy(dst, src, size) { verbatim_3i_0o(hex"5e", dst, src, size) }
+        
+          // Take most of the SSTORE cost before MCOPY. 
+          sstore(0, 1)
+        
+          // MCOPY using parameters from CALLDATA.
+          mcopy(calldataload(0), calldataload(32), calldataload(64))
+        
+          // Put MSIZE in storage.
+          sstore(0, msize())
```

Comment:
After this line, do

`mcopy(calldataload(0), calldataload(32), calldataload(64))`

again

Reply:
I will add one more test suite for it.

Reply:
Ok great :smile:

Reply:
Added in https://github.com/ethereum/tests/pull/1234.